examples/public_ip/README.md

```diff
@@ -0,0 +1,47 @@
+# Example: Public IP management for SDDC
+
+This is an example that demonstrates public IP management actions like creating and deleting public IP after SDDC has been created.
+
+To run the example:
+
+* Generate an API token using [VMware Cloud on AWS console] (https://vmc.vmware.com/console/)
+
+* Update the variables required parameters api_token and org_id in [variables.tf](https://github.com/terraform-providers/terraform-provider-vmc/blob/master/examples/public_ip/variables.tf) with your infrastructure settings. 
```

Comment:
remove either variables or parameters

Reply:
this comment applies to all README's updated in this PR

Reply:
Done